src/mlpack/bindings/go/CMakeLists.txt

```diff
@@ -43,6 +43,13 @@ if (BUILD_GO_BINDINGS)
   endif ()
 
   add_custom_target(go)
+
+  # Create model.go with package defination.
```

Comment:
Maybe it would be useful to note here that we put our build results into `src/mlpack.org/v1/mlpack/` so that setting `GOPATH` to the build directory works?

Reply:
Sorry I didn't get this. Is it regarding to add comment here?

Reply:
Yeah, I think it would be nice if you could add a comment about why we are putting builds results into the directory `src/mlpack.org/v1/mlpack/` instead of just `go/mlpack` or something. :+1:

Reply:
Updated this comment, You can take a look :).

Reply:
Looks great, thank you! :+1: